frontend/src/lib/i18n/locales/fr.json

```diff
@@ -78,7 +80,38 @@
         "practice": "Pratiquer",
         "error": "Erreur",
         "voting": "Vote",
-        "download": "Télécharger"
+        "download": "Télécharger",
+        "text": "Texte",
+        "order": "Ordre",
+        "normal": "Normal",
+        "totp": "https://fr.wikipedia.org/wiki/Mot_de_passe_%C3%A0_usage_unique_bas%C3%A9_sur_le_temps",
```

Comment:
@Dynnammo You can't put any links there, sorry

Reply:
Seen and corrected, was a mistake sorry